test/integration/golden_response/staging/get_triples/stats_var.json

```diff
@@ -334,17 +370,17 @@
       "objectTypes": [
         "Property"
       ],
-      "provenanceId": "dc/d7tbsb1"
+      "provenanceId": "dc/kyzlg31"
     },
     {
       "subjectId": "Count_Person_Female",
       "predicate": "provenance",
-      "objectId": "dc/d7tbsb1",
-      "objectName": "https://www.datacommons.org",
+      "objectId": "dc/kyzlg31",
+      "objectName": "https://www.census.gov/",
```

Comment:
this seems wrong. @pradh

Reply:
I see. This probably happens because we construct StatVar from StatPop/Obs, we might need some overrides to prefer the canonical stat-var source... I'll look into it.